proxy-node-shared/src/main/java/uk/gov/ida/notification/contracts/verifyserviceprovider/TranslatedHubResponse.java

```diff
@@ -50,6 +53,6 @@ public VspLevelOfAssurance getLevelOfAssurance() {
     }
 
     public Attributes getAttributes() {
-        return attributes;
+        return attributes == null ? null : fromNonMatchingAttributes(attributes);
```

Comment:
It feels like the non-null check should be moved to the `fromNonMatchingAttributes` method - and return null from there if `attributes` is null. Actually in that case `fromNonMatchingAttributes` should return an Optional as this is the proper way to deal with missing values

Reply:
I did wonder about that. Will update to an Optional.